send/ad_user_vsup

```diff
@@ -181,7 +181,7 @@ sub process_update() {
 			my $ad_entry = $ad_entries_map{$login};
 
 			# attrs without cn since it's part of DN to be updated
-			my @attrs = ('displayName','gecos','sn','givenName','mail','vsupPersonPersonalId','vsupPersonTitleHead','vsupPersonTitleTail','telephoneNumber','vsupPersonIdCardBarcode','vsupPersonIdCardChipNumber','altSecurityIdentities','accountExpires');
+			my @attrs = ('displayName','gecos','sn','givenName','mail','vsupPersonPersonalId','vsupPersonTitleHead','vsupPersonTitleTail','telephoneNumber','vsupPersonIdCardBarcode','vsupPersonIdCardChipNumber','altSecurityIdentities','accountExpires', 'userPrincipalName', 'proxyAddresses');
```

Comment:
I realized, that you need to add those new attributes to the list on line 69, where you load entries from AD. Otherwise they are not present in entry and comparison will result in always setting new value. This applies also to loading entry by login in  `get_entry_by_login()` method. Also it needs to be updated in both send scripts (normal + service users).

Reply:
I see. Added to both lists in both scripts.